airbyte-integrations/connectors/source-mongodb-internal-poc/src/main/java/io/airbyte/integrations/source/mongodb/internal/cdc/MongoDbCdcInitialSnapshotUtils.java

```diff
@@ -92,4 +112,27 @@ private static List<ConfiguredAirbyteStream> identifyStreamsToSnapshot(final Con
         .collect(Collectors.toList());
   }
 
+  private static void estimateInitialSnapshotSyncSize(final MongoClient mongoClient, final ConfiguredAirbyteStream stream) {
+    try {
+      final Document collectionStats = mongoClient.getDatabase(stream.getStream().getNamespace())
+          .runCommand(new BsonDocument("collStats", new BsonString(stream.getStream().getName())));
+
```

Comment:
Looks like [collStats](https://www.mongodb.com/docs/manual/reference/command/collStats/) has been deprecated in 6.2+.  Should we move this to use the [$collStats](https://www.mongodb.com/docs/manual/reference/operator/aggregation/collStats/#mongodb-pipeline-pipe.-collStats) instead?